libminifi/include/utils/file/PathUtils.h

```diff
@@ -62,6 +70,21 @@ inline bool isAbsolutePath(const char* const path) noexcept {
 #endif
 }
 
+inline utils::optional<std::string> canonicalize(const std::string &path) {
```

Comment:
it was in `FlowController::initializePaths` previously, its purpose is unclear to me

Reply:
I guess probably an oversight (someone writing one version did not realize the other version already exists), but this would require investigation and testing.  We can address this in another PR.